google/resource_dataproc_cluster.go

```diff
@@ -158,6 +159,14 @@ func resourceDataprocCluster() *schema.Resource {
 							Description: ` The name of the cloud storage bucket ultimately used to house the staging data for the cluster. If staging_bucket is specified, it will contain this value, otherwise it will be the auto generated name.`,
 						},
 
+						"temp_bucket": {
+							Type:         schema.TypeString,
+							Optional:     true,
```

Comment:
```suggestion
							Optional:     true,
							Computed:     true,
```

Reply:
Good find! I've updated it and also executed TestAccDataprocCluster_singleNodeCluster, TestAccDataprocCluster_withStagingBucket, and TestAccDataprocCluster_withTempBucket which all passed.

Reply:
great, glad to hear they pass locally now! I've upstreamed this change & the CI tests are running.